includes/edit-story.php

```diff
@@ -36,3 +36,34 @@
 <div id="edit-story">
 	<h1 class="loading-message"><?php esc_html_e( 'Please wait...', 'web-stories' ); ?></h1>
 </div>
+
+<script>
+(function() {
```

Comment:
What are these? I think we need to load these in a better way.

Reply:
Looks like debug tool to measure frames per seconds. Nothing that should be in the final version.